src/pybind/mgr/cephadm/templates/services/grafana/grafana.ini.j2

```diff
@@ -13,6 +13,10 @@
   http_port = {{ http_port }}
   http_addr = {{ http_addr }}
 [security]
+{% if not initial_admin_password %}
+  disable_initial_admin_creation = true
+{% else %}
   admin_user = admin
-  admin_password = admin
+  admin_password = {{ initial_admin_password }}
+{% endif %}
   allow_embedding = true
```

Comment:
BTW, according to the [PR I linked](https://github.com/rhcs-dashboard/ceph-dev/pull/46/files#), we need to add these settings under `[security]` section:
```
cookie_secure = true
cookie_samesite = none
```

And optionally disable the anonymous mode:
```
[auth.anonymous]
enabled = {{ grafana_anonymous_mode }}
```